apps/api/src/routes/v1_ratelimit_getOverride.ts

```
@@ -0,0 +1,83 @@
+    const override = await db.primary.query.ratelimitOverrides.findFirst({
+      where: (table, { eq, and }) =>
+        and(
+          eq(table.workspaceId, auth.authorizedWorkspaceId),
+          eq(table.identifier, identifier),
+        ),
+
+    });
+    if (!override) {
+      throw new UnkeyApiError({
+        code: "NOT_FOUND",
+        message: `Override not found`,
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)